src/main/java/org/broadinstitute/hellbender/tools/gvs/common/WeightedInterval.java

```diff
@@ -61,15 +61,15 @@ public WeightedInterval[] split(int basesInFirstInterval) {
                 this.getStart(),
                 this.getStart() + basesInFirstInterval - 1,
                 this.isNegativeStrand(),
-                this.getName() + "-1", // ensure names are unique
+                this.getName() == null ? null : this.getName() + "-1", // ensure non-null names are unique
```

Comment:
interval list files were getting lines like

```
chr1	772635	877632	+	null-2|null-1
```

which should have been
```
chr1	772635	877632	+	.|.
```

Reply:
nice catch---but are nulls okay here? Seems odd we were getting nulls to begin with?

Reply:
`WeightedSplitIntervals` only uses the two-arg version of the `WeightedInterval` constructor which means the `WeightedInterval`s it creates always have a `null` name.

Reply:
Would it make sense to give them names? Not necessary, but looks kind of odd without them.

Reply:
I would defer to others on that, I am new enough to this domain that this didn't jump out as odd to me. 🙂

Reply:
yeah I've always found the interval names in these files less than useful since unless your targeting biological constructs (ie gene names, etc) you typically end up doing something like "foo-XXX" which sort of is meaningless